### ISSUE TITLE
Selection into marking onto several pages

### ISSUE THREAD

**Issue by Artem-Sukhliak-Funzy** [kolyvan/kybook#398]
How it is possible to select and mark content that is on multiple pages? Couldn't figure out

**Comment by nicksolyom** [comment]
Agreed! Same experience here, I want to be able to drag my selections across pages
